### PR TITLE
corrected template roots error message

### PR DESCRIPTION
**Pull request by markfirmware**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export function warnIfMalformedTemplate(el, directive) {
     if (el.tagName.toLowerCase() !== 'template') {
         console.warn(`Alpine: [${directive}] directive should only be added to <template> tags. See https://github.com/alpinejs/alpine#${directive}`)
     } else if (el.content.childElementCount !== 1) {
-        console.warn(`Alpine: <template> tag with [${directive}] encountered with multiple element roots. Make sure <template> only has a single child element.`)
+        console.warn(`Alpine: template with [${directive}] encountered with unexpected number of root elements. Make sure <template> has a single child element. `)
```
Comment (HugoDF) [inline src/utils.js:31]: Any reason to remove the `<>`?
```suggestion
        console.warn(`Alpine: <template> tag with [${directive}] encountered with an unexpected number of root elements. Make sure <template> has a single child element. `)
```

Comment (markfirmware) [reply src/utils.js]: One more change coming ...k

File: src/utils.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export function warnIfMalformedTemplate(el, directive) {
     if (el.tagName.toLowerCase() !== 'template') {
         console.warn(`Alpine: [${directive}] directive should only be added to <template> tags. See https://github.com/alpinejs/alpine#${directive}`)
     } else if (el.content.childElementCount !== 1) {
-        console.warn(`Alpine: <template> tag with [${directive}] encountered with multiple element roots. Make sure <template> only has a single child element.`)
+        console.warn(`Alpine: <template> tag with [${directive}] encountered with an unexpected number of root elements. Make sure <template> has a single child element. `)
```
Comment (ryangjchandler) [inline src/utils.js:31]: Should `child` be changed to `root` here so that it's inline with the README? Or perhaps the README should be changed to `child`.

Comment (ryangjchandler) [inline src/utils.js:31]:
```suggestion
        console.warn(`Alpine: <template> tag with [${directive}] encountered with an unexpected number of root elements. Make sure <template> has a single root element. `)
```

**Review by ryangjchandler — COMMENTED** [review]
Just a small one, wondering whether the terminology here should be changed so that it matches the README.

**Comment by markfirmware** [comment]
> Just a small one, wondering whether the terminology here should be changed so that it matches the README.

I changed a bit of the readme to match the error message. Did I miss some?

**Comment by markfirmware** [comment]
> > Just a small one, wondering whether the terminology here should be changed so that it matches the README.
> 
> I changed a bit of the readme to match the error message. Did I miss some?

@ryangjchandler @HugoDF are these what you mean?

    gitpod /workspace/alpine $ grep -i root README.md
    | [`$el`](#el) |  Retrieve the root component DOM node. |
    > Note: `x-if` must have a single element root inside the `<template></template>` tag.
    > Note: `x-for` must have a single element root inside of the `<template></template>` tag.
    `$el` is a magic property that can be used to retrieve the root component DOM node.

**Comment by KevinBatdorf** [comment]
I think that comment came along with his suggested change just above it.

**Comment by markfirmware** [comment]
Yeah sorry I am having a little trouble navigating the gh presentation. Does this mean there are no pending actions for me to take?

**Comment by markfirmware** [comment]
@calebporzio ready to go. I wanted to compress to one commit but after thirty minutes of sesrching aimgavr up on that.

**Comment by calebporzio** [comment]
Thanks everyone!

**Pull request merged**